sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -298,6 +298,11 @@ type RawKeyValuePair = {
   Value: any;
 };
 
+/**
+ * @internal
+ * @ignore
+ * @interface InternalRawKeyValuePairs
```

Comment:
The `@interface` tag does not add any value in TS files. It was useful in JS where interfaces were not supported out of the box
```suggestion
```